tests/dtypes_test.py

```diff
@@ -0,0 +1,17 @@
+from commont import *
+
+
+def test_dtypes(ds_local):
+  assert ds.dtype(ds.x) == np.float64
+  assert ds.dtype(ds.y) == np.float64
+  assert ds.dtype(ds.m) == np.float64
+  assert ds.dtype(ds.nm) == np.float64
+  assert ds.dtype(ds.mi) == np.int64
+  assert ds.dtype('ints') == np.int64
+  assert ds.dtype('name') == 'S25'
```

Comment:
This seems inconsistent with the rest right

Reply:
Which one? the 'S25' type? Maybe I am wrong, but manual checks indicated that this is the dtype of the underlying array (or 'S|25' )

Reply:
Theses tests will never pass, Pandas will do the same, see https://stackoverflow.com/questions/26921836/correct-way-to-test-for-numpy-dtype